src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/SymbolKey/SymbolKey.cs

```diff
@@ -96,7 +95,7 @@ namespace Microsoft.CodeAnalysis
     /// <para>
     /// <see cref="SymbolKey"/>s are not guaranteed to work across different versions of Roslyn. They can be persisted
     /// in their <see cref="ToString()"/> form and used across sessions with the same version of Roslyn. However, future
-    /// versions may change the encoded format and may no longer be able to <see cref="Resolve"/> previous keys.  As
+    /// versions may change the encoded format and may no longer be able to <see cref="Resolve(Compilation, bool, bool, CancellationToken)"/> previous keys.  As
```

Comment:
To avoid updating a _heap_ of call sites, I added an overload rather than just one more optional parameter, so had to clarify here to avoid warnings.